Fix #5192, #4610 : [Developer Video] Understanding CI check failures & Replace wiki broken links (#5199)
oppia/oppia-android: wiki/Ktlint-Guide.md

- [How to fix the most common issues?](#how-to-fix-the-most-common-issues) 

# Installation

> Once you had completed all the [installation steps](https://github.com/oppia/oppia-android/wiki/Installing-Oppia-Android), you will be having a `ktlint` file in your `opensource/oppia-android-tools` folder. 

Reviewer: ```suggestion
Once you have completed all the [installation steps](https://github.com/oppia/oppia-android/wiki/Installing-Oppia-Android), you will have a `ktlint` file in your `opensource/oppia-android-tools` folder. 
```

Author: Done.
